Automattic/jetpack
# Refactor subscriptions code so it can be used in Fusion

Fixes a problem we've seen in developing the subscription block where we sometimes
forget to make the equivalent upstream changes.

#### Changes proposed in this Pull Request:

* Moves the class Jetpack_Subscriptions_Widget into /modules/subscriptions/views.php
* Moves the registration for the subscriptions block into /modules/subscriptions/views.php
* /modules/subscriptions/views.php will be shared with wordpress.com

#### Testing instructions:

- Get this branch running locally or on jurassic.ninja
- Ensure subscription widgets still function as usual
- add a widget to the blog
- add a subscription block to a post
- try subscribing from the front end

#### Proposed changelog entry for your changes:
No changelog needed
